data/registry_model/registry_proxy_model.py

```diff
@@ -346,13 +407,14 @@ def _update_manifest_for_tag(
         up_to_date = manifest.digest == upstream_digest
 
         placeholder = manifest.internal_manifest_bytes.as_unicode() == ""
+
         if up_to_date and not placeholder:
             return tag, False
 
         if upstream_manifest is None:
             upstream_manifest = self._pull_upstream_manifest(repo_ref.name, manifest_ref)
 
-        self._enforce_repository_quota(repo_ref)
+        # self._enforce_repository_quota(repo_ref)
```

Comment:
I think this is no longer called anywhere, if that's the case please remove it 💥